server/data/data-jmap-cassandra/pom.xml

```diff
@@ -79,6 +79,11 @@
             <artifactId>testing-base</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
wrong order, put it before `testing-base`

Reply:
okay, it's simple, so I squashed